python/ray/rllib/optimizers/apex_optimizer.py

```diff
@@ -163,12 +163,15 @@ def _init(
         self.learner = LearnerThread(self.local_evaluator)
         self.learner.start()
 
-        self.replay_actors = create_colocated(
-            ReplayActor,
-            [num_replay_buffer_shards, learning_starts, buffer_size,
-             train_batch_size, prioritized_replay_alpha,
-             prioritized_replay_beta, prioritized_replay_eps, clip_rewards],
-            num_replay_buffer_shards)
+        # TODO(ekl) use create_colocated() for these actors once
+        # https://github.com/ray-project/ray/issues/1734 is fixed
```

Comment:
Conveniently, the current behavior with cpu=None basically almost guarantees co-location.

Reply:
so we should revert this change after #1734?

Reply:
Maybe, it will require more testing.